cmd/syncthing/connections_tcp.go

```diff
@@ -98,8 +98,6 @@ func tcpListener(uri *url.URL, tlsCfg *tls.Config, conns chan<- model.Intermedia
 			continue
 		}
 
-		conns <- model.IntermediateConnection{
-			tc, model.ConnectionTypeDirectAccept,
-		}
+		conns <- relay.Connection{tc, relay.ConnectionTypeDirectAccept}
```

Comment:
:) What's in a name?

But yeah, this could perhaps better be

```go
conns <- DirectConnection{tc}
```

or something, where that type is always IsDirect()==true